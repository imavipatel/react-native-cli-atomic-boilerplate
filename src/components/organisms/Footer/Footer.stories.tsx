import * as React from 'react';
import { storiesOf } from '@storybook/react-native';
import { Footer } from './Footer';
import CenterView from '../../../../storybook/utils/CenterView/CenterView';

storiesOf('Footer', module)
  .addDecorator(story => <CenterView>{story()}</CenterView>)
  .add('default', () => <Footer />);
