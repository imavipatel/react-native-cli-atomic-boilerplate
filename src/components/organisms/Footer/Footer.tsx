import React from 'react';
import { View } from 'react-native';

import { styles } from './Footer.style';

export const Footer: React.FC<{}> = props => {
  return <View style={styles.container} />;
};
