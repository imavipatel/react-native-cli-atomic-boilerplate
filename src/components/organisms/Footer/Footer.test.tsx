import React from 'react';
import { render } from '@testing-library/react-native';
import { Footer } from './Footer';

describe('Footer', () => {
  const spyOnPress = jest.fn();

  beforeEach(() => {
    jest.clearAllMocks();
  });

  it('should render correctly', () => {
    const tree = render(<Footer />);

    expect(tree).toMatchSnapshot();
  });
});
