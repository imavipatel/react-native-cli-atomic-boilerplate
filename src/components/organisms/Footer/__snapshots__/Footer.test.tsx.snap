// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<View
  style={
    Object {
      "flex": 4,
    }
  }
/>
`;
